src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1286,6 +1303,7 @@ impl DdlController {
                                 dropping_sink_id: Some(sink_id),
                                 updated_sink_catalogs: vec![],
                             },
+                            None, // release_ctx is None because we already drop the objects and not require atomic drop things when replacing table.
```

Comment:
I don't get what does "not require atomic drop things when replacing table" mean here :eyes:

I think it's just "no source is dropped since we are dropping a sink into table"